Bottom: 0a8223ef8800adfa73beebe3f5f56863ed9143c4
Top:    5a7662d466053913a4bfe767f8aeb41e48e282dd
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-17 12:42:19 +0800

Refresh of autofs-5.0.7-use-ulimit-max-open-files-if-greater-than-internal-maximum.patch

---

diff --git a/daemon/automount.c b/daemon/automount.c
index 5a48d1d..2bbffb0 100644
--- a/daemon/automount.c
+++ b/daemon/automount.c
@@ -2109,7 +2109,7 @@ int main(int argc, char *argv[])
 	if (res == -1 || rlim.rlim_max <= MAX_OPEN_FILES)  {
 		rlim.rlim_cur = MAX_OPEN_FILES;
 		rlim.rlim_max = MAX_OPEN_FILES;
-	}	
+	}
 	res = setrlimit(RLIMIT_NOFILE, &rlim);
 	if (res)
 		printf("%s: can't increase open file limit - continuing",
